File: generators/java/src/main/java/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/ResponseMeasure.java
package dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Getter;
import lombok.ToString;

@Getter
@ToString
public class ResponseMeasure {

    @JsonProperty("response_time")
    private String responseTime;
}

File: src/main/kotlin/io/github/dqualizer/dqlang/archive/loadtesttranslator/dqlang/modeling/ResponseMeasure.kt
package io.github.dqualizer.dqlang.archive.loadtesttranslator.dqlang.modeling

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Getter
import lombok.ToString

@Getter
@ToString
class ResponseMeasure {
    @JsonProperty("response_time")
    val responseTime: String? = null
}
